packages/admin/test/e2e/cli.feature.ts

```diff
@@ -0,0 +1,515 @@
+import { AirnodeRrp, AirnodeRrpFactory } from '@airnode/protocol';
+import { ethers } from 'ethers';
+import * as admin from '../../src';
+import { execSync } from 'child_process';
+import { existsSync } from 'fs';
+import { difference } from 'lodash';
+
+const PROVIDER_URL = 'http://127.0.0.1:8545/';
+const CLI_EXECUTABLE = `${__dirname}/../../dist/cli.js`;
+
+describe('CLI', () => {
+  let provider: ethers.providers.JsonRpcProvider;
+  let deployer: ethers.providers.JsonRpcSigner;
+  const aliceDerivationPath = "m/44'/60'/0'/0/1";
+  let alice: ethers.Wallet;
+  const bobDerivationPath = "m/44'/60'/0'/0/2";
+  let bob: ethers.Wallet;
+  let airnodeRrp: AirnodeRrp;
+  // https://hardhat.org/hardhat-network/#hardhat-network-initial-state
+  const mnemonic = 'test test test test test test test test test test test junk';
```

Comment:
> Are there no issues with re-using the same mnemonic between tests? Ideally we want each test to run with a unique mnemonic. 

This is special hardhat mnemonic - all funded accounts by hardhat are derived from this mnemonic (hence the link above).
We need it because we need the get the accounts which later used in tests.
It's true that the accounts will be still the same, but we deploy a different `AirnodeRrp` contract every time.

> There might be weird cases at some point where a test depends on something happening before it, especially if it's altering the blockchain

Not sure what you mean, when you `await` some contract call, you can be sure it is included on the blockchain. And having multiple `AirnodeRrp` doesn't matter. Once you "forget" it's address it's like having a fresh blockchain.

Reply:
In the node E2E tests, I needed to use a unique mnemonic Airnode instance in each test. Because they're in different files, the order they get executed isn't guaranteed. They each go through the process of submitting API calls or withdrawals so they can't rely on the same mnemonic.

This isn't actually a problem though - I see you're using this already for tests:

```ts
const airnodeWallet = ethers.Wallet.createRandom().connect(provider);
const airnodeMnemonic = airnodeWallet.mnemonic.phrase;
```